source/base.c

```diff
@@ -348,6 +348,33 @@ bool C2D_DrawTriangle(float x0, float y0, u32 clr0, float x1, float y1, u32 clr1
 	return true;
 }
 
+bool C2D_DrawLine(float x0, float y0, u32 clr0, float x1, float y1, u32 clr1, float thickness, float depth)
+{
+	C2Di_Context* ctx = C2Di_GetContext();
+	if (!(ctx->flags & C2DiF_Active))
+		return false;
+	if (6 > (ctx->vtxBufSize - ctx->vtxBufPos))
+		return false;
+
+	float dx = x1-x0, dy = y1-y0, len = sqrt(dx*dx+dy*dy), th = thickness/2;
```

Comment:
Should be `sqrtf`. `sqrt` takes a double.

Reply:
ah thanks, didn't catch that. Fixed in latest commit.